src/utility-types.ts

```diff
@@ -42,7 +43,7 @@ export interface MessageValue<
   /**
    * Kill code.
    */
-  readonly kill?: number
+  readonly kill?: KillBehavior | number
```

Comment:
You can enforce the type to KillBehavior.

Reply:
We still use a number when we send the message back to the worker to clean the interval

Reply:
> @pioardi I'm currently improving this in the background, give me some minutes!
> 
> branch `issue-70-shini-improvements-1`

oh yeah, now I recognize you :P